### ISSUE TITLE
Cache previous/next slide in presentation mode

### ISSUE THREAD

**Issue by owickstrom** [mpflanzer/splitshow#51]
SplitShow Version 0.9.8 (1)
OSX Version 10.11.6

When I move through slides _the first time_ in presentation mode, i.e. the first time I visit each slide, it takes a few milliseconds to render it, and meanwhile, there's a white screen. The effect becomes a white "flash".

If I browse through all the slides _in presentation mode_, they seem to get cached for later, and I can browse back to the first and start presenting from there, without any flashes.

I have no idea how this is implemented, but should there perhaps be a similar caching of the next slide in presentation mode?

Thanks for SplitShow, BTW! I'm using it for all my presentations.

**Comment by mpflanzer** [comment]
Thanks for the feedback. Good to hear that SplitShow is indeed used.

I definitely have seen the effect that you're describing but currently I am not able to reproduce it anymore. I think it is most likely that Apple changed something internally (I have updated to version 10.12.2) as I did not add any caching of my own. It is completely done in Apple's PDF view.
But to be sure could you check if you still have the same issue when using the latest version of SplitShow? That should be 0.9.8 (4).
If it still happens for you I will have a look if there is a way to trigger the internal caching manually before the slide is actually loaded.

**Comment by owickstrom** [comment]
So I just upgraded to the new version as I have upgraded my laptop to macOS Sierra. Thanks for providing support!

The flashing problem seems to have disappeared, as you say. I'll verify this on projectors/external displays soon as well, but running the presentation view in fullscreen gives no flashing effect now, from what I can tell. It does however jump, or offset, slightly on each slide change. It almost looks like an antialiasing thing that kicks in. I do not think this is a problem for me, though.

If you want to close this issue that's fine by me. :)
